web/client-backplane/src/main/java/io/deephaven/javascript/proto/dhinternal/io/deephaven/barrage/flatbuf/barrage_generated/io/deephaven/barrage/flatbuf/BarrageSnapshotOptions.java

```diff
@@ -15,13 +15,16 @@ public class BarrageSnapshotOptions {
     public static native void addColumnConversionMode(
             Builder builder, int columnConversionMode);
 
+    public static native void addMaxMessageSize(Builder builder, double maxMessageSize);
```

Comment:
Were these flatbuffer changes made manually, or were they generated? (I don't know if you are aware these are generated, but require extra steps.)

Reply:
@niloc132 generated these since I was having trouble on my M1 dev machine